scripts/app.js

```diff
@@ -31,7 +34,7 @@
       $.getJSON('/data/projects.json', function(rawData){
         Project.loadAll(rawData);
         localStorage.setItem('rawData', JSON.stringify(rawData));
-        next();
+        ProjectView.initIndexPage();
       });
     }
   };
```

Comment:
Good idea!  You should be using the functional programming methods that we learned.  Like .map instead of the for loop here.